themes/bootstrap3/js/notifications.js

```diff
@@ -0,0 +1,140 @@
+                type = 'page';
+            }
+            if ($(this).data('broadcast-id') !== undefined && $(this).data('broadcast-id') !== '') {
+                order.push($(this).data('broadcast-id'));
+                type = 'broadcast';
+            }
+        });
+
+        $.ajax({
+            url : '/vufind/AJAX/JSON?method=NotificationsReorder',
+            dataType:'json',
+            data: {
+                'order': order,
+                'type': type,
+            },
+            success : function(data) {
+            },
+        });
+    });
+
+    $('.notifications-visibility').on('click', function(e){
+        e.preventDefault();
+        var $this = $(this);
+        $.ajax({
+            url : '/vufind/AJAX/JSON?method=NotificationsVisibility',
+            dataType:'json',
+            data: {
+                'page-id': $this.parents('tr').data('page-id'),
+                'broadcast-id': $this.parents('tr').data('broadcast-id'),
+                'visibility': $this.data('visibility'),
+            },
+            success : function(data) {
+                if (data.data.visibility == 1) {
+                    $this.find('.icon--font').removeClass('fa-eye-slash');
+                    $this.find('.icon--font').addClass('fa-eye');
+                    $this.data('visibility', '0');
+                } else {
+                    $this.find('.icon--font').removeClass('fa-eye');
+                    $this.find('.icon--font').addClass('fa-eye-slash');
+                    $this.data('visibility', '1');
+                }
+            },
+        });
+    });
+
+    $('.notifications-visibility-global').on('click', function(e){
+        e.preventDefault();
+        var $this = $(this);
+        $.ajax({
+            url : '/vufind/AJAX/JSON?method=NotificationsVisibility',
+            dataType:'json',
+            data: {
+                'page-id': $this.parents('tr').data('page-id'),
+                'broadcast-id': $this.parents('tr').data('broadcast-id'),
+                'visibility-global': $this.data('visibility-global'),
+            },
+            success : function(data) {
+                if (data.data.visibility_global == 1) {
+                    $this.find('.icon--font').removeClass('fa-globe');
+                    $this.find('.icon--font').addClass('fa-times-circle');
+                    $this.data('visibility-global', '0');
+                } else {
+                    $this.find('.icon--font').removeClass('fa-times-circle');
+                    $this.find('.icon--font').addClass('fa-globe');
+                    $this.data('visibility-global', '1');
+                }
+            },
+        });
+    });
+
+    function isExternalUrl () {
+        if ($('input[type="checkbox"][name="is_external_url"]').prop('checked')) {
+            return true;
+        }
+        return false;
+    }
+
+    function toggleFormExternalUrl () {
+        /*$('.is_external_url_fields').each(function(){
```

Comment:
Is this code intentionally commented out, or should it be removed entirely?

Also note that continuous integration is complaining about some style issues in this file -- we'll need to clean those up to get this merged, but of course if you prefer to finalize the logic before applying polish, that is fine for now.

Reply:
I removed the code, as it was only a first approach used during development.